test/lit.cfg

```diff
@@ -945,8 +945,9 @@ def source_compiler_rt_libs(path):
                                         if lib.startswith('libclang_rt.')
                                         and config.compiler_rt_platform in lib])
 
-source_compiler_rt_libs(make_path(test_resource_dir, 'clang', 'lib',
-                        platform.system().lower()))
+compiler_rt_dir = make_path(test_resource_dir, 'clang', 'lib',
```

Comment:
Would it be possible to add a comment to explain the why here?

Reply:
LOL! I need to add a comment to explain my request for a comment!
My thought was to add comments to this file to explain what is going on w.r.t. the new feature being added. (I know nothing about lit.cfg files and thought maybe there were breadcrumbs you could add.)

Reply:
This feels like another case where there's not a reasonable comment. This part didn't change, and the part below builds a path and tests if it exists.